local-dev/telemetry-pipeline/docker-compose.yml

```
@@ -0,0 +1,160 @@
+# this one is for local development only; in production refer under the deployment 
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)